### PR TITLE
fix: typos in the code

### PR DESCRIPTION
**Pull request by kavinkuma6**
### Describe what this PR does
- Fixes https://github.com/dailydotdev/daily/issues/960
- Fixes typos throughout the code

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [x] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/icons/Bell/Notify/filled.svg
Comment (rebelchris) [inline packages/shared/src/components/icons/Bell/Notify/filled.svg:1]: What changed in the icons?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Thanks a lot for these fixes!
Just wondering what changed in the icon cases?

File: packages/shared/src/components/icons/Bell/Notify/filled.svg
Comment (kavinkuma6) [reply packages/shared/src/components/icons/Bell/Notify/filled.svg:1]: @rebelchris `id` has typo. Not a big deal though.
Screenshot 2023-08-18 at 4 39 09 PM

**Comment by kavinkuma6** [comment]
@rebelchris also shall I add a pre commit hook to prevent typos in future? I think it is worth doing it.

File: packages/shared/src/components/icons/Bell/Notify/filled.svg
Comment (rebelchris) [reply packages/shared/src/components/icons/Bell/Notify/filled.svg:1]: Ah I see 🙏

**Review by rebelchris — APPROVED** [review]
Looks good to me, @sshanzel can you also review and potentially merge 🙏

As for your question, we don't really leverage pre-commit at the moment.
Perhaps if we can add it as a linting rule that would be cool. (Separate PR)

**Review by sshanzel — APPROVED** [review]
The build passed too! Looks great to me. Thank you so much for your contribution 🙏

**Pull request merged**